certify-service/src/test/resources/application-test.properties

```diff
@@ -25,6 +25,7 @@ mosip.certify.authn.jwk-set-uri=${mosipbox.public.url}${server.servlet.path}/oau
 mosip.certify.authn.allowed-audiences={ '${mosipbox.public.url}${server.servlet.path}/issuance/credential' }
 
 mosip.certify.supported.jwt-proof-alg={'RS256'}
+mosip.certify.well-known.file.uri=https://raw.githubusercontent.com/mosip/inji-config/develop/certify-mock-openid4vci-wk.json
```

Comment:
For unit test case, we cannot reach out to external system, we should figure out a way to use the local file.

Reply:
If I change this URL to a local file, it'll have to be a URI to my fork(vharsh/inji-certify), it cannot be pointing to mosip/inji-certify as the file doesn't exist yet. If I go ahead with the latter, the `CertifyApplicationTest` will start to fail.

Reply:
@vharsh we can have this json file in resources folder in test directory

Reply:
@swatigoel It's already present in this PR but if I add it's reference here the build will fail.
There're two possible solution for this:
1. I set it to point to my fork --> https://github.com/vharsh/inji-certify/blob/injicert-248-add/certify-service/src/main/resources/openid4vci-wk.json and later update it to point to mosip/inji-certify
2. I point it to the config repo for the time being and update it to point to this repo

If I point the location to the same branch while the file doesn't exist, the maven test for this PR will fail, resulting in my PR not getting merged.

Reply:
The suggestion is not to point to github URL at all, for unit test cases, even if the internet is unavailable, we should be able to run unit test case successfully.
Ideally we should point to a local file directory using a URL schema like file:/// ***

Reply:
I'm not able to use `file:///` scheme to get a response, I get `Target host not specified` or `Unroutable protocol scheme file://`.

Reply:
resolved